jsonschema/src/keywords/format.rs

```diff
@@ -465,6 +477,7 @@ pub(crate) fn compile<'a>(
             #[cfg(feature = "draft201909")]
             "uuid" if draft_version == Draft::Draft201909 => Some(UUIDValidator::compile(context)),
             "uri" => Some(URIValidator::compile(context)),
+            "duration" => Some(DurationValidator::compile(context)),
```

Comment:
Could you please put it behind a feature flag and a condition similar to the uuid format above?

Reply:
OK, another commit has been added